doc/jobs/doc_jobs.rst

```diff
@@ -31,6 +31,14 @@ Each *doc build files* can have one of three specific documentation types:
   honor an environment variable identifying the destination of the generated
   documentation.
 
+* the **docker_build** type also operates on independent repositories.
+  For each *doc build file* with that type a Jenkins job is created.
+  Each repository must provide its own Dockerfile below `docker/image/`,
```

Comment:
It'd be great to add an example to the docs. I opted not to permalink it to keep the example up-to-date in theory.

```suggestion
  Each repository must provide its own Dockerfile below `docker/image/` (Example: [ros2/rosindex](https://github.com/ros2/rosindex/tree/ros2/docker/image)),
```